frontend/app/room/page.tsx

```diff
@@ -1,13 +1,11 @@
 import { getRooms } from "@/app/lib/actions";
-import RoomCard from "@/app/ui/room/card";
+import { redirect } from "next/navigation";
 
 export default async function RoomListPage() {
-  const rooms = await getRooms();
-  return (
-    <div className="flex flex-wrap gap-8">
-      {rooms.map((room, index) => (
-        <RoomCard room={room} key={index} />
-      ))}
-    </div>
-  );
+  const rooms = (await getRooms({ joined: true })) || [];
+  if (rooms.length === 0) {
+    return <div className="text-center">No rooms found.</div>;
+  } else {
+    redirect("/room/" + rooms[0].id);
+  }
```

Comment:
Look at this document. `await` is not necessary because `redirect` is not async function.
https://nextjs.org/docs/app/api-reference/functions/redirect